src/gui/folderwizard/folderwizard.cpp

```diff
@@ -13,90 +13,102 @@
  */
 
 #include "folderwizard.h"
+
+#include "ui_folderwizardsourcepage.h"
+#include "ui_folderwizardtargetpage.h"
+
 #include "account.h"
-#include "accountstate.h"
 #include "common/asserts.h"
 #include "configfile.h"
 #include "creds/abstractcredentials.h"
-#include "folderman.h"
 #include "networkjobs.h"
-#include "ocwizard_deprecated.h"
-#include "selectivesyncdialog.h"
 #include "theme.h"
 
+#include "gui/accountstate.h"
+#include "gui/folderman.h"
+#include "gui/ocwizard_deprecated.h"
+#include "gui/selectivesyncdialog.h"
+
+#include <QCheckBox>
 #include <QDesktopServices>
 #include <QDir>
+#include <QEvent>
 #include <QFileDialog>
-#include <QFileInfo>
 #include <QFileIconProvider>
+#include <QFileInfo>
 #include <QInputDialog>
+#include <QMessageBox>
+#include <QTreeWidget>
 #include <QUrl>
+#include <QVBoxLayout>
 #include <QValidator>
 #include <QWizardPage>
-#include <QTreeWidget>
-#include <QVBoxLayout>
-#include <QEvent>
-#include <QCheckBox>
-#include <QMessageBox>
 
 #include <stdlib.h>
 
-namespace OCC {
-
-Q_LOGGING_CATEGORY(lcFolderWizard, "gui.folderwizard", QtInfoMsg)
-
-QString FormatWarningsWizardPage::formatWarnings(const QStringList &warnings, bool isError) const
+namespace {
+QString formatWarnings(const QStringList &warnings, bool isError = false)
 {
     QString ret;
     if (warnings.count() == 1) {
-        ret = isError ? tr("<b>Error:</b> %1").arg(warnings.first()) : tr("<b>Warning:</b> %1").arg(warnings.first());
+        ret = isError ? QApplication::translate("FolderWizard", "<b>Error:</b> %1").arg(warnings.first()) : QApplication::translate("FolderWizard", "<b>Warning:</b> %1").arg(warnings.first());
```

Comment:
Doesn't this change the context of these messages, i.e., invalidate old translations?